pkg/runner/runner.go

```diff
@@ -92,6 +94,15 @@ func Run(cfg *Config) error {
 		grpcServer := grpc.NewServer()
 		otlpgrpc.RegisterTracesServer(grpcServer, api.NewTraceServer(client))
 
+		queryPlugin := shared.StorageGRPCPlugin{
+			Impl: jaegerquery.New(client.Connection),
+		}
+		err := queryPlugin.GRPCServer(nil, grpcServer)
+		if err != nil {
+			log.Error("msg", "Creating jaeger query GRPC server failed", "err", err)
+			return err
```

Comment:
Seeing lines around this, to be consistent, maybe not return err? Also makes the logs more clear.

Reply:
Everything is already consistent the top-level func return error, sub-go-routines don't